.azure/azure-pipelines.yml

```diff
@@ -49,10 +52,7 @@ stages:
 
 
       - job: Build_And_Test
```

Comment:
I think this should be called "Build"

Reply:
https://github.com/graycoreio/daffodil/pull/1721#discussion_r681743786